src/api/types/helpers/index.ts

```diff
@@ -1,4 +1,4 @@
 export interface MapYesNoFieldParams {
-  answer?: boolean;
+  answer?: boolean | null;
```

Comment:
do you need the | null if it;s `answer?`

Reply:
yes, without this, when calling `mapYesNoField` with an answer that is null, TypeScript complains because it's not a boolean.